src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -972,12 +973,13 @@ public function softDeletes($column = 'deleted_at', $precision = 0)
     /**
      * Add a "deleted at" timestampTz for the table.
      *
+     * @param  string  $column
      * @param  int  $precision
      * @return \Illuminate\Support\Fluent
      */
-    public function softDeletesTz($precision = 0)
+    public function softDeletesTz($column = 'deleted_at', $precision = 0)
```

Comment:
I think this one breaks backward compatibility, but not sure.

I will rebase it if you say so

Reply:
Please, move $column = 'deleted_at' to the second argument

Reply:
yes, it does. so you'll have to target 'master', which it looks like your are